src/DGtal/dec/DiscreteExteriorCalculus.h

```diff
@@ -294,14 +316,43 @@ namespace DGtal
     eraseCell(const Cell& cell);
 
     /**
+     * Update indexes for all cells.
+     * Cell insertion order == index may not be preserved.
+     */
+    void
+    updateIndexes();
+
+    /**
      * Get all cells properties.
      * @return associative container from Cell to Property.
      */
-    Properties
+    const Properties&
     getProperties() const;
 
     /**
-     * Identity operator from order-forms to order-forms.
+     * Get all signed cells with specific @a order and @a duality in index order.
+     * @tparam order order of signed cells.
+     * @tparam duality duality of signed cells.
+     * @return index ordered signed cells.
+     */
+    template <Order order, Duality duality>
+    const SCells&
+    getIndexedSCells() const;
+
+    /**
+     * Reorder operator from <em>order</em>-forms to <em>order</em>-forms.
```

Comment:
i can't get the @e command to work on something like (order-1)-form...
doxygen doc says "To emphasize multiple words use <em>multiple words</em>." without limiting its use to html or latex output.